api/dao/containerutil.py

```diff
@@ -200,11 +201,11 @@ def pluralize(cont_name):
         return SINGULAR_TO_PLURAL[cont_name]
     elif cont_name in PLURAL_TO_SINGULAR:
         return cont_name
-    raise Exception('Could not pluralize unknown container name {}'.format(cont_name))
+    raise InputValidationException('Could not pluralize unknown container name {}'.format(cont_name))
```

Comment:
I would rather we keep this as an exception and try/catch this in the handler layer. If code gets to containerutil.py (or anything in `api.dao`) we expect it has already been validated*. The handler can then throw InputValidationException. 

*This might not be 100% true ATM but is a goal I'd like to achieve